chirp/drivers/ft450d.py

```diff
@@ -29,9 +30,16 @@
 
 LOG = logging.getLogger(__name__)
 
-CMD_ACK = 0x06
-# TBD: Enable some form of generated UI field, for the memory tags
-# That field wiould not be stored in img file, but generated in get_memory
+HAS_FUTURE = True
+try:                         # PY3 compliance
+    from builtins import bytes
+    from past.builtins import range     # RJD 12/18/22

```

Comment:
Why are you importing this? This makes this code python3-uncompliant. If you're using range below and need a list, then you should make it a list.